includes/validation/class-amp-validation-manager.php

```diff
@@ -119,6 +119,7 @@ class AMP_Validation_Manager {
 	 *
 	 * Keys are post IDs and values are whether the post has been re-validated.
 	 *
+	 * @deprecated In 2.1 the classic editor block validation was removed.
 	 * @var bool[]
 	 */
 	public static $posts_pending_frontend_validation = [];
```

Comment:
This variable can now be removed, no? Unless it was being referenced directly by themes/plugins.

Reply:
I can see that [two plugins](https://wpdirectory.net/search/01F15WMZ27G17VKG9KVQ6WS4K8) are making use of this variable, but in those cases they rely on the AMP plugin as a library so removing this variable shouldn't affect them.

Reply:
It can't be removed because it's being used by this mini plugin: https://gist.github.com/westonruter/31ac0e056b8b1278c98f8a9f548fcc1a

Reply:
Oh I see. Maybe we could add a note about that also so it isn't accidentally removed in the future.

Reply:
Added